docs/users.md

```diff
@@ -314,6 +316,83 @@ To create a control plane of type `host` run the command:
 kflex create cp4 --type host
 ```
 
+To create a control plane of type `external` with the required options, run the command: 
+
+```shell
+kflex adopt --adopted-context <kubeconfig-context-of-external-cluster> cp5
+```
+
+*Important*: This command generates a secret containing a long-lived token for accessing 
+the external cluster within the namespace associated with the control plane. The secret is automatically 
+removed when the associated control plane is deleted.
+
+### Creating a control plane of type `external` with the API
+
+To create a control plane of type `external` with the API, you need to provide 
+first a **bootstrap secret** containing a bootstrap Kubeconfig for accessing the external cluster.
+The bootstrap Kubeconfig is used by the KubeFlex controllers to generate a long-lived
+token for accessing the external cluster.  The bootstrap kubeconfig is required to have only one context, 
+so given a Kubeconfig for the external cluster `$EXTERNAL_KUBECONFIG` with context for the external
+cluster `$EXTERNAL_CONTEXT` you can generate the `$BOOTSTRAP_KUBECONFIG` with the command:
+
+```shell
+kubectl --kubeconfig=$EXTERNAL_KUBECONFIG config view --minify --flatten \
+--context=$EXTERNAL_CONTEXT > $BOOTSTRAP_KUBECONFIG
+```
+
+If your external cluster is on the same host machine as the KubeFlex hosting cluster, you
```

Comment:
If I understand correctly, the critical issue is not "same machine" but rather using the loopback address? (Someone who has already successfully wrangled domain names as needed should not be told they have to do additional work here.)

Reply:
good point. Fixed.